docs/scenario_guides/scenario_secret_passing.rst

```diff
@@ -0,0 +1,32 @@
+.. _secret_passing:
+
+Passing Secrets
+===============
+
+Sometimes it may be useful to use `build secrets <https://docs.docker.com/build/building/secrets/>`_ when
+creating an Execution Environment. This can be done with a combination of the use of :ref:`additional_build_steps`
+within the EE definition file, and the :ref:`extra-build-cli-args` CLI option.
```

Comment:
```suggestion
When creating an Execution Environment, it might be useful to use `build secrets <https://docs.docker.com/build/building/secrets/>`_. This can be done with a combination of the use of :ref:`additional_build_steps`
within the EE definition file, and the :ref:`extra-build-cli-args` CLI option.
```

Just rephrase, not a blocker.